plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/LocalDockerNode.java

```diff
@@ -33,7 +33,7 @@
     private String host;
 
     @Inject
-    public LocalDockerNode(@Named("vfs.local.fs_root_dir") String projectsFolder) throws IOException {
+    public LocalDockerNode(@Named("host.projects.folder") String projectsFolder) throws IOException {
```

Comment:
We aggree that on machine we have "che.projects.root"
may be we should rename this constant to "host.projects.root" or something like this